app/client/packages/design-system/widgets/src/components/Modal/src/ModalFooter.tsx

```diff
@@ -6,7 +6,12 @@ import { Button } from "../../Button";
 import type { ModalFooterProps } from "./types";
 
 export const ModalFooter = (props: ModalFooterProps) => {
-  const { closeText = "Close", onSubmit, submitText = "Submit" } = props;
+  const {
+    allowInteraction = true,
```

Comment:
```suggestion
    excludeFromTabOrder = true,
```

I think having one name for things that do the same thing is a good idea.